platform/android/java/lib/src/org/godotengine/godot/input/GodotGestureHandler.java

```diff
@@ -70,15 +73,17 @@ public void onLongPress(MotionEvent event) {
 		//Log.i("GodotGesture", "onLongPress");
 	}
 
+	@RequiresApi(api = Build.VERSION_CODES.M)
 	@Override
 	public boolean onDoubleTap(MotionEvent event) {
 		//Log.i("GodotGesture", "onDoubleTap");
 		final int x = Math.round(event.getX());
 		final int y = Math.round(event.getY());
+		final int button = event.getSource() == InputDevice.SOURCE_MOUSE ? MotionEvent.BUTTON_PRIMARY : 0; // for now any double click with mouse counts as primery click
```

Comment:
Doesn't seem necessary.

Reply:
Need to know if I want to send doubletap event as pressed to godot, which in android I found out is sends it with pressed false, while the editor expect pressed to be true to track doubletap events